controller/src/test/java/io/pravega/controller/fault/ControllerClusterListenerTest.java

```diff
@@ -52,13 +62,9 @@
 
 
     @Before
-    public void setup() {
+    public void setup() throws Exception {
```

Comment:
We could handle thrown exception in one or more of the following ways.
1. Log the exception and stack trace,
2. Return from the method,
3. Rethrow the exception (possibly after transforming it)
1 and 2 are problematic in test case, because build output does not typically contain test logs and we lose stack traces and other exception details that may be crucial for identifying root cause of intermittent test failures. Throwing exception from the method at least shows us the exception type and stack trace in build logs.